### ISSUE TITLE
Specify size of each sprite

### ISSUE THREAD

**Issue by sagron** [ForkandBeard/Alferd-Spritesheet-Unpacker#13]
I have a spritesheet where each of the sprites is exactly 64x64, I would like to specify that so the alignment is correct on the exported images (instead of clipping to the visible portion)

**Comment by imukai** [comment]
Being able to specify a universal size of the sprites on the sheet should be a basic feature and should make your processing a lot easier to calculate rather than finding edges and guessing where sprites are.  

Is the project dead?

**Comment by ForkandBeard** [comment]
ASU's main function is unpacking sprite sheets where the size of each individual sprite is irregular and or unknown. If the size and location of all the sprites are known, I'm not sure why you'd need to use ASU?

I might add it in the future, but I'm not sure how to do that without making the UI more complex for ASU's main function, of unpacking sprites of unknown size and location.

**Comment by imukai** [comment]
Fair enough, if that's your purpose.  Your main page just says "ASU is a simple spriting tool which takes an animation spritesheet image and automatically turns each sprite into an individual image." which is what I was searching for.  That being a program to reliably cut up a spritesheet into individual files.  Most spritesheets I've used are of uniform size, including this one that I wanted to cut up that had 400 sprites on it.   I'll find other means though.  Thank you for the quick reply and sorry to bother you.

**Comment by ForkandBeard** [comment]
Oh ok, yeah I think the description then is probably a bit misleading, I'll try and think of a better description.
